plugins/tiddlywiki/tiddlyweb/save-wiki-button.tid

```diff
@@ -0,0 +1,44 @@
+title: $:/core/ui/Buttons/save-wiki
+tags: $:/tags/PageControls
+caption: {{$:/core/images/cloud-sync}} Server status
+description: Status of synchronisation with server
+
```

Comment:
It would be nice, if you would use `\whitespace trim` and indent the code properly. So it would be a 100 times more readable and I think much easier to understand. 

The second advantage would be, that there are no unwanted <p> tags, that are a horror for styling.

Reply:
Hi @pmario I'll give it a go.